### PR TITLE
fix: issues with subpath and missing string literals

### PR DESCRIPTION
**Pull request by lukas-reining**
## This PR

@toddbaert These are the things I had to change to get the demo running on K8s 1.25 on Docker Desktop.
Also It was complaining about some string literals missing.

### Related Issues

Fixes #1234523

### CODE REVIEW

File: deployment/collector.yaml [hunk 1]
```diff
@@ -85,4 +85,4 @@ data:
         metrics:
           receivers: [ otlp ]
           processors: [ batch ]
-          exporters: [ prometheus ]
\ No newline at end of file
+          exporters: [ prometheus ]
```
Comment (beeme1mr) [inline deployment/collector.yaml:88]: What's the change here?

**Comment by toddbaert** [comment]
I will test this on my end and see if everything still seems to work well.

File: deployment/collector.yaml (hunk 1 above)
Comment (toddbaert) [reply deployment/collector.yaml:88]: I think this is just a line-ending change.

Comment (lukas-reining) [reply deployment/collector.yaml:88]: Ya, think so too, just happened when formatting.

**Review by toddbaert — APPROVED** [review]
Confirmed this works for me with `kind`. Thanks a lot @lukas-reining

**Pull request merged**